### PR TITLE
[11.0] [ADD] Add module Romanian Accounting Report.

### PR DESCRIPTION
**Pull request by feketemihai**
The module includes Trial Balance (from Account Group + Account Account) - to have hierarcy

### CODE REVIEW

File: l10n_ro_report_trial_balance/static/src/js/l10n_ro_report_trial_balance_backend.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+odoo.define('l10n_ro_report_trial_balance.l10n_ro_report_trial_balance_backend', function (require) {
+'use strict';
+
+var core = require('web.core');
+var Widget = require('web.Widget');
+var ControlPanelMixin = require('web.ControlPanelMixin');
+var session = require('web.session');
+var ReportWidget = require('l10n_ro_report_trial_balance.l10n_ro_report_trial_balance');
```
Comment (dhongu) [inline l10n_ro_report_trial_balance/static/src/js/l10n_ro_report_trial_balance_backend.js]: card ca trebuie l10n_ro_report_trial_balance.l10n_ro_report_trial_balance_widget

File: l10n_ro_report_trial_balance/report/trial_balance.py [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+    def _compute_account_group_values(self):
+        if not self.account_ids:
+            acc_res = self.line_account_ids
+            groups = self.env['account.group'].search(
+                [('code_prefix', '!=', False)])
+            for group in groups:
+                accounts = acc_res.filtered(
+                    lambda a: a.account_id.id in
+                    group.compute_account_ids.ids)
```
Comment (dhongu) [inline l10n_ro_report_trial_balance/report/trial_balance.py:228]: compute_account_ids  nu exista in account.group

Comment (feketemihai) [reply l10n_ro_report_trial_balance/report/trial_balance.py:228]: Yep, pentru widget, compute_account_ids exista aici, https://github.com/feketemihai/account-financial-reporting/blob/e38193495b18e4e9f52112b083ae2286cc71cb98/account_financial_report/models/account_group.py#L22, si modulul depinde de account_financial_report ca sa nu adaug de 2 ori campu lrespectiv. Cel mai bine ar fii sa facem review in Engleza atat timp cat suntem putini, si o sa depinde de Maintainers..

File: l10n_ro_report_trial_balance/wizards/trial_balance_wizard.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    def onchange_date_range_id(self):
+        """Handle date range change."""
+        self.date_from = self.date_range_id.date_start
+        self.date_to = self.date_range_id.date_end
+
+    @api.multi
+    def button_export_html(self):
+        self.ensure_one()
+        action = self.env.ref(
+            'l10n_ro_report_trial_balance.l10n_ro_report_trial_balance')
```
Comment (dhongu) [inline l10n_ro_report_trial_balance/wizards/trial_balance_wizard.py]: action = self.env.ref( 'l10n_ro_report_trial_balance.action_l10n_ro_report_trial_balance')
Correct is action_l10n_ro_report_trial_balance

Comment (feketemihai) [reply l10n_ro_report_trial_balance/wizards/trial_balance_wizard.py]: Nu prea are importanta, daca nu e setat ia el implicit numele meodului din care face parte, daca da ia toata calea..

Comment (dhongu) [reply l10n_ro_report_trial_balance/wizards/trial_balance_wizard.py]: Eu zic ca e gresit numele actiunii.

File: l10n_ro_report_trial_balance/report/trial_balance.py [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+                if self.hide_account_balance_at_0:
+                    accounts = accounts.filtered(
+                        lambda a: a.balance_debit != 0 or
```
Comment (dhongu) [inline l10n_ro_report_trial_balance/report/trial_balance.py]: In ce modul e definit balance_debit?

Comment (dhongu) [reply l10n_ro_report_trial_balance/report/trial_balance.py]: a.balance_debit este de fapt a.debit_balance

**Comment by feketemihai** [comment]
@dhongu Ai patit asa ceva cu ANAF, https://travis-ci.org/OCA/l10n-romania/jobs/353809849#L1495, la mine pe local nu am nici o eroare

**Pull request merged**